app/include/zmk/split/bluetooth/service.h

```diff
@@ -21,4 +31,7 @@ struct zmk_split_run_behavior_payload {
 } __packed;
 
 int zmk_split_bt_position_pressed(uint8_t position);
-int zmk_split_bt_position_released(uint8_t position);
\ No newline at end of file
+int zmk_split_bt_position_released(uint8_t position);
+int zmk_split_bt_sensor_triggered(uint8_t sensor_number,
```

Comment:
Minor: `sensor_event` uses `sensor_index`, while this uses `sensor_number`. Are these the same thing? If so, making the name consistent would be good.

This also appears to be all the same data as a `sensor_event`. Could you just pass a pointer to that struct instead?

Reply:
So.... `struct sensor_event` was intended as a *data exchange* format, which is why it is packed. I'm *hesitant* to have a data exchange struct also be used for public API like this. Thoughts?